london_cafes/london_cafes/templates/index.html

```diff
@@ -0,0 +1,8 @@
+{% load static %}
+
+{% block main %}
+    <div id="content">
```

Comment:
please remove id, since we have no css yet no need for classes, But also generally avoid using IDs since we will be using Object oriented CSS and BEM.